core/src/main/scala/com/dimafeng/testcontainers/FixedHostPortGenericContainer.scala

```diff
@@ -8,6 +8,7 @@ class FixedHostPortGenericContainer(imageName: String,
                                     env: Map[String, String] = Map(),
                                     command: Seq[String] = Seq(),
                                     classpathResourceMapping: Seq[(String, String, BindMode)] = Seq(),
+                                    fileSystemBind: Seq[(String, String, BindMode)] = Seq(),
```

Comment:
would you mind moving this argument to be last? it'll help to protect those who use positional arguments